Homework4/hw4_2.sql

```diff
@@ -0,0 +1,147 @@
+drop database if exists shop;
+create database shop;
+use shop;
+
+
+create table customers(
+id int primary key auto_increment,
+customer_name varchar(30) not null,
+customer_surname varchar(30),
+town_id int not null
+);
+
+create table orders(
+id int primary key auto_increment,
+order_name_id int not null,
+category_id int not null,
+quality_id int not null,
+factory_id int not null,
+garanty_id int
+);
+
+create table order_names(
```

Comment:
Order name sounds odd, the better name just `products`.
Also, it would suit if we relate `qualities ` to this table.
The same as `factories ` 😉